client/src/lib/dependency-injection/tests/ReducerContainer-test.js

```diff
@@ -1,6 +1,6 @@
 /* global jest, jasmine, describe, beforeEach, it, pit, expect, process */
 
-jest.mock('../MiddlewareRegistry', () => () => ({ add: jest.fn() }));
+jest.mock('../MiddlewareRegistry', () => jest.fn().mockImplementation(() => ({ add: () => {} })));
```

Comment:
This was necessary to fix an error where the mocked class wasn't being returned correctly. The error was along the lines of `MiddlewareRegistry is not a constructor`